cl/recap/mergers.py

```diff
@@ -1792,6 +1792,12 @@ async def merge_attachment_page_data(
         if not all(sanity_checks):
             continue
 
+        # Missing on some restricted docs (see Juriscraper)
+        if "page_count" in attachment and attachment["page_count"] is None:
+            # Attachment 0 may not have page count since it is the main rd.
+            if attachment["attachment_number"] != 0:
+                continue
+
```

Comment:
I think this can be simplified as:

```suggestion
       if "page_count" in attachment and attachment["page_count"] is None and attachment["attachment_number"] != 0:
    continue

```